shell/probe/raindrop.ps1

```diff
@@ -6,9 +6,18 @@ function Execute {
     try {
         $stdoutTempFile = New-Item -path "$dir\stdout.log" -Force
         $stderrTempFile = New-Item -path "$dir\stderr.log" -Force
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru -RedirectStandardOutput $stdoutTempFile -RedirectStandardError $stderrTempFile).ExitCode
-        $Code = if (Test-Path $File) {$R} Else {127}
-        return $Code
+        $proc = Start-Process -FilePath $File -NoNewWindow -PassThru -RedirectStandardOutput $stdoutTempFile -RedirectStandardError $stderrTempFile
+
+        $errVar = $null
```

Comment:
nit: `timeoutErr` is clearer

Reply:
I will acquiesce to that once we are sure you can't get messages that aren't timeouts.